packages/login/src/utils/authApi.ts

```diff
@@ -74,12 +74,29 @@ export interface ITokenResponse {
   token: string
 }
 
+const loginClientVersion = APP_VERSION
+
 export function request<T>(options: AxiosRequestConfig) {
   const onSuccess = (response: AxiosResponse<T>) => {
+    const gatewayVersion = response.headers['x-version']
+
+    if (gatewayVersion && gatewayVersion !== loginClientVersion) {
+      console.log(
+        `Version Mismatch: Frontend is running on ${loginClientVersion}, whereas backend is running on ${gatewayVersion}. Please Reload to get the latest client`
+      )
+      throw new AxiosError('Version_MisMatch')
```

Comment:
Can we use `VERSION_MISMATCH` instead? It's a bit more standard capitalisation for a constant error identifier like this